R/ids_get.R

```diff
@@ -1,79 +1,119 @@
-#' Fetch Debt Statistics from the World Bank International Debt Statistics API
+#' Fetch Data from the World Bank International Debt Statistics (IDS) API
 #'
-#' This function returns a tibble with debt statistics data fetched from the
-#' World Bank International Debt Statistics (IDS) API. The data can be filtered
-#' by geographies, series, counterparts, and time periods.
+#' Retrieves standardized debt statistics from the World Bank's International
+#' Debt Statistics (IDS) database, which provides comprehensive data on the
+#' external debt of low and middle-income countries. The function handles
+#' country identification, data validation, and unit standardization, making it
+#' easier to conduct cross-country debt analysis and monitoring.
 #'
-#' @param geographies A character vector representing the geographic codes
-#'  (e.g., "ZMB" for Zambia). This argument is required and cannot contain NA
-#'  values.
-#' @param series A character vector representing the series codes (e.g.,
-#'  "DT.DOD.DPPG.CD"). This argument is required and cannot contain NA values.
-#' @param counterparts A character vector representing counterpart areas (e.g.,
-#'  "all", "001"). This argument is required and cannot contain NA values
-#'  (default: "all").
-#' @param start_date An optional numeric value representing the starting year
-#'  (e.g., 2015). It must be greater than or equal to 1970. If not provided, the
-#'  entire time range is used.
-#' @param end_date An optional numeric value representing the ending year (e.g.,
-#'  2020). It must be greater than or equal to 1970 and cannot be earlier than
-#'  `start_date`. If not provided, the entire available time range is used.
-#' @param progress A logical value indicating whether to display a progress
-#'  message during the request process (default: `FALSE`). Must be either `TRUE`
-#'  or `FALSE`.
+#' @param geographies A character vector of geography identifiers representing
+#'   debtor countries and aggregates. Must use `geography_id` from
+#'   `ids_list_geographies()`:
```

Comment:
If you use \link{ids_list_geographies} instead of `ids_list_geographies()`, then you get a hyperlink to the function in the docs and I believe this is desired :)